src/plugins/extra/heading_anchors.rs

```diff
@@ -0,0 +1,148 @@
+use slug::slugify;
+
+use crate::{
+    parser::{core::CoreRule, inline::Text},
+    plugins::cmark::{
+        block::heading::ATXHeading,
+        inline::{
+            backticks::CodeInline,
+            emphasis::{Em, Strong},
+            link::Link,
+        },
+    },
+    MarkdownIt, Node, NodeValue, Renderer,
+};
+
+use super::strikethrough::Strikethrough;
```

Comment:
inconsistent import style, this uses `super::`, and there are `crate::` imports above